src/components/App.js

```diff
@@ -1,15 +1,20 @@
 import React, { Component } from 'react';
 import './App.css';
-import Routes from '../Routes';
+import Root from "../containers/Root";
 
 class App extends Component {
   render() {
     return (
-      <div className="App container">
-        <Routes />
+      <div className="App">
+        <header className="App-header">
+          <p>
+            [In Development]
+          </p>
+          <Root>document.getElementById("root")</Root>
+        </header>
```

Comment:
Will merging this PR remove the routes that we set up in React?

Reply:
I'm going to remove this. I was just using the Root component for testing purposes. I'll pull down my Redux pull requests, merge them into one branch along with the current master branch, and push that. The front-end should reflect exactly what's in the master branch then.

Reply:
Sounds good!